packages/http-client-csharp/generator/TestProjects/Spector/http/type/union/src/Generated/Models/GetResponseProp3.cs

```diff
@@ -2,33 +2,15 @@
 
 #nullable disable
 
-using System;
-using System.ComponentModel;
-
 namespace _Type.Union
 {
-    public readonly partial struct GetResponseProp3 : IEquatable<GetResponseProp3>
+    public enum GetResponseProp3
```

Comment:
This seems unexpected. Was the spec changed?

Reply:
The content did not really change, just renamed due to order of anonymous models change.

Reply:
So, this one and `packages/http-client-csharp/generator/TestProjects/Spector/http/type/union/src/Generated/Models/GetResponseProp1.cs` exchanged their content.